app/models/rankinglist.rb

```diff
@@ -0,0 +1,3 @@
+class Rankinglist < Event
+  validates :discipline, :deadline, :startdate, :enddate, :player_type, presence: false
```

Comment:
Just a conceptual question: Does Playertype truely not matter here? Because if it doesn't we should maybe talk about removing the field all together for consitency sake. I'm not sure if a League and Tournament have to have this field. Haven't talked to POs about this yet.
Also why is discipline not needed for a ranking list? Isn't a rankinglist for just one discipline (like chess for example) aswell? Otherwise against we need to remove that one from the events validations and put all these validations into the right models.

Reply:
I know that my PO Pius @GittiHab likes to have a selection in the create event form for single player/team events. That doesn't necessarily mean we need to store that in the database, though. If we start working with min players per team and max players per team fields, we could infer if it is a single player event, if min players and max players are 1. But yeah, that's something we need to reach a conclusion on.

Discipline should probably be available for ranking lists, though.